modules/apps/object/object-test/src/testFunctional/tests/ObjectAudit.testcase

```diff
@@ -90,6 +90,90 @@ definition {
 		}
 	}
 
+	@description = "LPS-164582 - Verify CRUD functionality with audit when using object definition entry that has the entry history activated"
+	@priority = 5
+	test AssertCRUDWithObjectEntry {
+		property portal.acceptance = "true";
+
+		task ("Given a custom object definition with toogle for Entry History in Audit Framework activated and a custom field") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 193633",
+				objectName = "CustomObject193633",
+				pluralLabelName = "Custom Objects 193633");
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "Text",
+				fieldLabelName = "Custom Field",
+				fieldName = "customField",
+				fieldType = "String",
+				isRequired = "false",
+				objectName = "CustomObject193633");
+
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object 193633");
+
+			Check.checkToggleSwitch(locator1 = "ObjectPortlet#ENABLE_ENTRY_HISTORY_IN_AUDIT_FRAMEWORK_BUTTON");
+
+			ObjectAdmin.publishObjectViaUI();
+		}
+
+		task ("and Given a custom object entry added") {
+			ObjectAdmin.addObjectSingleFieldEntryViaAPI(
+				fieldName = "customField",
+				objectName = "CustomObject193633",
+				value = "New object entry");
+		}
+
+		task ("and Given a custom object entry updated") {
+			ObjectAdmin.goToCustomObject(objectName = "CustomObject193633");
+
+			ObjectPortlet.updateEntry(
+				entry = "New object entry",
+				updateEntry = "New object entry updated");
+		}
+
+		task ("and Given a custom object entry deleted") {
+			ObjectAdmin.deleteObjectEntryViaAPI(
+				fieldName = "customField",
+				objectName = "CustomObject193633",
+				value = "New object entry updated");
+		}
+
+		task ("When accessing the audit admin") {
+			Audit.openAuditAdmin();
+		}
+
+		task ("Then assert that the audit can see the custom object entry created, updated and deleted") {
```

Comment:
Combine this:

`Then the Add, Update, and Delete events appear in the Audit records`